websites/D/DscJobs.org/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "DscJobs.org",
```

Comment:
DscJobs is their official name or am I wrong? So change it and change folder name too.